pkg/hessian_protocol.go

```diff
@@ -0,0 +1,46 @@
+package hessian2
+
+import (
+	"github.com/cloudwego/kitex/pkg/remote"
+	"sync"
+)
+
+// must be strict read & strict write
+var (
+	bpPool sync.Pool
+	_      BaseProtocol = (*BinaryProtocol)(nil)
+)
+
+func newBP() interface{} {
```

Comment:
Not referenced?
You might need to add an `init()` with `bpPool.New = newBP`;

Reply:
done